crates/handler/src/evm.rs

```diff
@@ -43,20 +43,20 @@ pub trait EvmTr {
 }
 
 /// Execute EVM transactions. Main trait for transaction execution.
-pub trait ExecuteEvm: ContextSetters {
+pub trait ExecuteEvm {
```

Comment:
ContextSetters are part of API, idea is that you create one EVM and you can execute multiple transactions on top of it. That is why you need a Trait that knows the type of Transaction and Block.

The step forward for this would be to bring `Tx/Block` types to `ExecuteEvm` trait.

Reply:
Thanks. It seems like most of the changes included in this PR are covered in #2195 and #2194. Should anything else be preserved?

Reply:
it seems that's it